recipes/libpqxx/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,11 @@
+#include <cstdlib>
+#include <iostream>
+
+#include "pqxx/pqxx"
+
+int main()
+{
+    std::cout << "libpqxx version: " << PQXX_VERSION << std::endl;
```

Comment:
You're not actually calling any pqxx function here: you're just printing a macro.
To test the package, some functions inside the libraries must be called.

You can use the example at https://libpqxx.readthedocs.io/en/stable/a01329.html.
You can hardcode (or pass) it a bad connection string and if the `broken_connection` exception is thrown, just print `error thrown` and return 0 because that was expected.

Reply:
Update tests: c1c40eb5557529725455c4825a4afe735f6ba3f4